tests/common.py

```diff
@@ -34,6 +34,28 @@ def get(self, endpoint_url: str, token: str | None = None):
             endpoint_url, HTTP_AUTHORIZATION=f"Bearer {access_token}"
         )
 
+    def post(self, endpoint_url: str, data: Any, token: str | None = None):
```

Comment:
This is already implemented in `develop` branch.

Reply:
This is not finished yet, thanks for Your checks :) good eyes